README.md

```diff
@@ -75,7 +75,17 @@ assertSnapshot(matching: vc, as: .image(on: .iPadMini(.portrait)))
 assertSnapshot(matching: vc, as: .recursiveDescription(on: .iPadMini(.portrait)))
 ```
 
-> ⚠️ Warning: Snapshots must be compared using a simulator with the same OS, device gamut, and scale as the simulator that originally took the reference to avoid discrepancies between images.
+> ⚠️ Warning: Snapshots may differ slightly unless compared on the same OS,
+> device gamut, and scale as the simulator that originally took the reference.
+> If this cannot be avoided, acceptance in differences can be configured by
+> setting the `subpixelThreshold`-parameter.
+>
+> Example:
+> ```swift
+> // Allow each subpixel to deviate up to 5 byte-values
+> assertSnapshot(matching: vc, as: .image(on: .iPhoneX, subpixelThreshold: 5))
```

Comment:
@Deco354 this fork doesn't seem to include said `subpixelThreshold` param?

Reply:
It took me a while to find this myself but it is there. The parameter is inside several snapshot test strategies such as [Sources/SnapshotTesting/Snapshotting/UIView.swift](https://github.com/tumblr/swift-snapshot-testing/pull/1/files#diff-06746e4dfaad5a7b5f90fe231ea8a3d91e45d881cbce7eacdd6d3085d19cd660) and is used like so:

`assertSnapshot(matching: view, as: image(subpixelThreshold: UInt8(2) ))`

Just to be sure I've checked to see the diff is identical to the [original PR](https://github.com/pointfreeco/swift-snapshot-testing/pull/571/files) and I just gave it a test run to check it works on the Tumblr iOS app and with a subpixel threadhold value of `2` our intel recorded tests that were previously failing on M1 devices are passing now.

Reply:
@Deco354 ah many apologies I read the line as: 
```swift
assertSnapshot(matching: vc, as: .image(on: .iPhoneX), subpixelThreshold: 5)
```
🤦🏻‍♂️

Reply:
thanks, these changes are working for us now!